packages/document/docs/en/guide/debug/rsdoctor.mdx

```diff
@@ -0,0 +1,49 @@
+# Use Rsdoctor
+
+[Rsdoctor](https://rsdoctor.dev/) is a build analyzer that can visually display the build process, such as compilation time, code changes before and after compilation, module reference relationships, duplicate modules, etc.
+
+If you need to debug the build outputs or build process, you can use Rsdoctor for troubleshooting.
+
+## Quick Start
+
+In a Rsbuild-based project, you can enable Rsdoctor as follows:
+
+1. Install the Rsdoctor plugin:
+
+import { PackageManagerTabs } from '@theme';
+
+<PackageManagerTabs command="add @rsdoctor/rspack-plugin -D" />
+
+2. Add `RSDOCTOR=true` env variable before the CLI command:
+
+```bash
+# dev
+RSDOCTOR=true rsbuild dev
+
+# build
+RSDOCTOR=true rsbuild build
+```
+
+After running the above commands, Rsbuild will automatically register the Rsdoctor plugin, and after the build is completed, it will open the build analysis page. For complete features, please refer to [Rsdoctor document](https://rsdoctor.dev/).
+
+## Options
+
+If you need to configure the [options](https://rsdoctor.dev/config/options) provided by the Rsdoctor plugin, please manually register the Rsdoctor plugin:
+
```

Comment:
I think it would be better if the Rsdoctor plugin supports reading the `rsdoctor.config.ts` configuration file?

Reply:
Rsdoctor options is not very complex, it is just like other webpack plugins, so we may not need to provide a separate config file for it.